### DISCUSSION TITLE
The application cannot be launched using a regular account

### DISCUSSION THREAD

**Discussion by Umbrella-TEC** [Maplespe/DWMBlurGlass#168] [Q&A]
Runs as an administrator, does not run under a regular account (the folder with the application is on the desktop)

No errors occur, log is empty

**Comment by Maplespe** [comment]
The program cannot be placed in any user folder, including the desktop.
> Note that due to DWM is running in a sandbox, this application must be placed in a non-user directory, Please place it in another directory.

Anyway, this is due to a permission restriction on dwm.exe and has nothing to do with DWMBlurGlass.

**Comment by Umbrella-TEC** [comment]
It also does not launch from the program files and windows folder. It looks like transparency won't work in a regular account.

**Comment by Maplespe** [comment]
The account must have Administrator level privileges, and the user must have Administrators access even if you are running as Administrator.
Also, multiple simultaneous user logins are not supported at this time, so if you are already logged in as one user and log out to log in as another, DWMBlurGlass will not automatically run on the other user.
